projects/media-viewer/src/lib/toolbar/toolbar-event.service.ts

```diff
@@ -6,7 +6,7 @@ export type HighlightMode = boolean;
 export type DrawMode = boolean;
 
 @Injectable({providedIn: 'root'})
```

Comment:
As you’ve moved these as Providers in the Toolbar Module, providedIn: root is a bit conflicting, as it will no longer use the root injector. Might be worth removing.

Reply:
Are you sure? I thought it was natural for multiple classes to use the root injector... I will google it

Reply:
tbh, not 100% but i believe the providedIn is a short-cut… not sure which one would take precedence.